sweep/sweeper.go

```diff
@@ -79,6 +79,10 @@ type PendingInput struct {
 	// map it into a fee rate whenever we attempt to cluster inputs for a
 	// sweep.
 	FeePreference FeePreference
+
+	// LastFeeRate is the most recent fee rate used for this input within a
+	// transaction broadcast to the network.
+	LastFeeRate lnwallet.SatPerKWeight
```

Comment:
Had to add this field in order to display fee rates on the RPC for inputs with confirmation targets. Is it worth setting this value to something once the input comes in to the sweeper to not have it be 0? The definitive fee rate won't be known until we actually try to sweep the input.

Reply:
I think 0 is okay to signal that it hasn't been in a tx yet.

Reply:
Sounds good, added a note to explain the behavior.